### PR TITLE
Prepare directory structure for V1

### PR DESCRIPTION
**Pull request by decentralgabe**

### CODE REVIEW

**Comment by dtmcg** [comment]
do you want to keep a working copy under spec/spec.md?

**Comment by bumblefudge** [comment]
Some thoughts, each of which I'd recommend converting to new issues for addressing post v1 unless they have easy answers or can be addressed by quick edits:
1. why `author` and not, say, `controller`?
2. I took a kind of contrarian, sassy stab at the "todo" to define trust topics, please iterate if it's too sassy or judgmental. Spec is a little cryptic without a little more explanation of its use-case and mental model!
3. The Topic Registry TODO I think should be done before v1.  Since topics are defined as URIs of schemas, you could just point to [schema-directory](https://github.com/decentralized-identity/schema-directory/blob/main/content/schemas.md) or something equivalent at, say, TOIP?  
3B. Sidenote: the schema directory doesn't really look like the [other DIF registries](https://identity.foundation/claim-format-registry/#registry), maybe we can give that group a nudge to make something more... presentable? It feels like another case of the "v1 can't be cut if it depends on a v0 spec or registry" problem that has plagued this WG since time immemorial 😉

**Comment by bumblefudge** [comment]
also note
image

File: spec/v1/spec.md [hunk 1]
```diff
@@ -503,25 +501,33 @@ This document specifies neither _entity_ nor _topic_ at the document level, and
...
+3. Is hosting this document with a single party a risk: to availablility, privacy erosion, or censorship in some manner?
+4. If I intend for this document to be replicated, how do I do that? Is there a way to incentivize other parties to replicate my documents?
 
 ## Appendix
 
 ### Topic Registry
 
 ::: todo Topic Registry
   Create a place for topics to be registered.
+  The [schema directory](https://github.com/decentralized-identity/schema-directory/blob/main/content/schemas.md) seems like one place where schemas are registered, but I would also recommend adding a definition of what you mean by "registry" and ideally more than one example. __chair
```
Comment (bumblefudge) [inline spec/v1/spec.md:530]: This should probably be addressed before v1; see PR discussion for more detail

**Review by bumblefudge — CHANGES_REQUESTED** [review]

**Comment by decentralgabe** [comment]
Thanks @bumblefudge for the review.

I opened up https://github.com/decentralized-identity/schema-directory/issues/5 on the Schema Directory repo.

I've added spec text around Trust Topics that I hope addresses your feedback.

I wonder about the semantic different between _author_ and _controller_. I believe _author_ implies _controller_ -- or at least that is my intention, though I'm not sure the inverse is true. The intent was to establish who created the document.

Got rid of those TODOs. Let me know what you think.

File: spec/v1/spec.md [hunk 2]
```diff
@@ -527,8 +522,14 @@ There may be different motivations for interacting with [[ref:Trust Establishmen
 
 ### Topic Registry
 
-::: todo Topic Registry
-  Create a place for topics to be registered.
+As per the section on [Publication & Discovery](#publication-discovery) there are a number of possibilities for hosting [[ref:Trust Establishment Documents]]. A related concept is defining a set of well-known [[ref:Topics]] that can find utility across an ecosystem of trust. One such option in housing these well-defined [[ref:Topics]] is a _Topic Registry_. Topic Registries can be as simple as [GitHub repositories](https://github.com/) that host a collection of topics or a website in a similar fashion to [Schema.org](https://schema.org/).
```
Comment (decentralgabe) [inline spec/v1/spec.md:525]: @csuwildcat let me know what you think about this section, specifically on the accuracy of the DWN text

File: spec/v1/spec.md [hunk 3]
```diff
@@ -527,8 +522,14 @@ There may be different motivations for interacting with [[ref:Trust Establishmen
...
+
+In the [DIF](https://identity.foundation), there are already some analagous registries such as the work item [Schema Directory](https://github.com/decentralized-identity/schema-directory/) which aims to host commonly used schemas such as those for defining Employment or KYC. It is possible to extend this registry to include _Trust Topics_, or create a new project if there is enough interest.
+
+Another project at DIF, [Decentralized Web Nodes](https://identity.foundation/decentralized-web-node/spec/), also provide the ability to host and address multiple types of data. Taking advantage of the [Collections interface](https://identity.foundation/decentralized-web-node/spec/#collections) it is possible to create a _Trust Topic Collection_ that is exposed via [Feature Detection](https://identity.foundation/decentralized-web-node/spec/#feature-detection), and given a public read [permission](https://identity.foundation/decentralized-web-node/spec/#permissions). This would allow anyone to interrogate a DWN, or network of DWNs, as to whether they have a Trust Topic Registry, and next, which Trust Topics they have access to.
+
+::: todo
```
Comment (decentralgabe) [inline spec/v1/spec.md:531]: @TelegramSam could use your magic here

**Pull request closed without merging**